spec/std/isa/csr/mseccfg.yaml

```diff
@@ -10,9 +10,107 @@ long_name: Machine Security Configuration
+            with executable privileges. Executing code from a region without a matching rule or with a matching S/U-mode-only rule is denied.
+
+        d.  If mseccfg.MML is not set, the combination of `pmpcfg.RW=01` remains reserved for future standard use.
+
+      The truth table when the `mseccfg.MML` is set:
+
+      [cols="4*^.^1,2*^.^3", separator="!", %autowidth, options="header"]
+      !====
+      4+^! Bits on _pmpcfg_ register 2+^! Result
+      ! L ! R ! W ! X ! M Mode ! S/U Mode
+
+      ! 0 ! 0 ! 0 ! 0 2+^! Inaccessible region (Access Exception)
+      ! 0 ! 0 ! 0 ! 1 ! Access Exception ! Execute-only region
+      ! 0 ! 0 ! 1 ! 0 2+^! Shared data region: Read/write on M mode, Read-only on S/U mode
+      ! 0 ! 0 ! 1 ! 1 2+^! Shared data region: Read/write for both M and S/U mode
+      ! 0 ! 1 ! 0 ! 0 ! Access Exception ! Read-only region
+      ! 0 ! 1 ! 0 ! 1 ! Access Exception ! Read/Execute region
+      ! 0 ! 1 ! 1 ! 0 ! Access Exception ! Read/Write region
+      ! 0 ! 1 ! 1 ! 1 ! Access Exception ! Read/Write/Execute region
+      ! 1 ! 0 ! 0 ! 0 2+^! Locked inaccessible region* (Access Exception)
+      ! 1 ! 0 ! 0 ! 1 ! Locked Execute-only region* ! Access Exception
+      ! 1 ! 0 ! 1 ! 0 2+^! Locked Shared code region: Execute only on both M and S/U mode.*
+      ! 1 ! 0 ! 1 ! 1 2+^! Locked Shared code region: Execute only on S/U mode, read/execute on M mode.*
+      ! 1 ! 1 ! 0 ! 0 ! Locked Read-only region* ! Access Exception
+      ! 1 ! 1 ! 0 ! 1 ! Locked Read/Execute region* ! Access Exception
+      ! 1 ! 1 ! 1 ! 0 ! Locked Read/Write region* ! Access Exception
+      ! 1 ! 1 ! 1 ! 1 2+^! Locked Shared data region: Read only on both M and S/U mode.*
+      !====
+
+      *Locked rules cannot be removed or modified until a PMP reset, unless mseccfg.RLB is set.
+    type(): |
+      if (MSECCFG_MML_TYPE == "read-only-0" || MSECCFG_MML_TYPE == "read-only-1") {
+        return CsrFieldType::RO;
+      } else if (MSECCFG_MML_TYPE == "sticky") {
+        return CsrFieldType::RW-R; // restricted: 0→1 allowed, 1→0 not allowed
+      }
+    sw_write(csr_value): return csr_value.MML | CSR[mseccfg].MML;
+    reset_value: UNDEFINED_LEGAL
+  MMWP:
+    location: 1
+    definedBy: Smepmp
+    description: |
+      Machine-Mode Allowlist Policy (mseccfg.MMWP) is a sticky bit, meaning that once set it cannot be unset until a PMP reset. When set it
+      changes the default PMP policy for M-mode when accessing memory regions that don't have a matching PMP rule, to denied instead of ignored.
+    type(): |
+      if (MSECCFG_MML_TYPE == "read-only-0" || MSECCFG_MML_TYPE == "read-only-1") {
+        return CsrFieldType::RO;
+      } else if (MSECCFG_MML_TYPE == "sticky") {
+        return CsrFieldType::RW-R; // restricted: 0→1 allowed, 1→0 not allowed
+      }
+    sw_write(csr_value): return csr_value.MMWP | CSR[mseccfg].MMWP;
+    reset_value: UNDEFINED_LEGAL
```

Comment:
This should depend on MSECCFG_MML_TYPE:
- read-only-0: 0
- read-only-1: 1
- sticky: 0